src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.ts

```diff
@@ -37,140 +36,21 @@ export class GrafanaComponent implements OnInit, OnChanges {
   uid: string;
 
   docsUrl: string;
+  settingGrafana: any;
```

Comment:
It seems that 'settingGrafana' is not used anywhere.

Reply:
ack